storage/pebble/registers.go

```diff
@@ -0,0 +1,40 @@
+package pebble
+
+import (
+	"fmt"
+	"path"
+
+	"github.com/cockroachdb/pebble"
+	"go.uber.org/multierr"
+
+	"github.com/onflow/flow-go/storage/pebble/registers/payload"
+)
+
+// library that implements pebble storage for registers
+type Registers struct {
+	Payloads *payload.Payloads
```

Comment:
We should rename payloads into registers, because we are not storing payloads, but actually registers

Reply:
This module is such a thin wrap, I'm not sure if we actually need it? 

I think we can move the actual DB operation to this module.